packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -108,6 +111,27 @@ const COLUMNS_VM_PV = [
       return sr !== undefined && sr.name_label
     },
   },
+  {
+    itemRenderer: (vdi, { hosts, pool, srs }) => {
+      const sr = srs[vdi.$SR]
+      const containerId = sr !== undefined && sr.$container
+      const container = containerId === pool.id ? pool : hosts[containerId]
+      return (
+        container && (
+          <Link to={`${container.type}s/${container.id}`}>
+            {renderXoItem(container)}
+          </Link>
+        )
+      )
+    },
+    name: _('vdiSrContainer'),
+    sortCriteria: (vdi, { hosts, pool, srs }) => {
+      const sr = srs[vdi.$SR]
+      const containerId = sr !== undefined && sr.$container
+      const container = containerId === pool.id ? pool : hosts[containerId]
+      return container !== undefined && container.name_label
```

Comment:
Would it be easier to build a `containerBySr` map once and for all and pass it via the `userData`?